telemetry/opting-out-of-telemetry.adoc

```diff
@@ -14,12 +14,21 @@ On {product-title}, Telemetry is always enabled. You cannot opt out.
 endif::[]
 
 ifdef::openshift-enterprise,openshift-origin[]
-.Prerequisites
 
-* Install the OpenShift Command-line Interface (CLI), commonly known as `oc`.
-* You must log in to the cluster with a user that has the `cluster-admin` role.
-* You need to have the `cluster-monitoring-config` ConfigMap object with the `data/config.yaml` section. See xref:../monitoring/cluster-monitoring/configuring-the-monitoring-stack.adoc#configuring-the-monitoring-stack[Creating cluster monitoring ConfigMap] for details.
+To opt out of Telemetry, you must:
+
+. xref:../telemetry/opting-out-of-telemetry.html#insights-operator-new-pull-secret_telemetry[Modify the global cluster pull secret] to disable Telemetry and the Insights Operator.
```

Comment:
Same here - these xrefs are basically duplicates of the toc, and they need to use .adoc if you keep them.